### DISCUSSION TITLE
"Event in progress"  - in perpetuity... :(

### DISCUSSION THREAD

**Discussion by FrigateEnthusiast** [blakeblackshear/frigate#6432] [Ask A Question] [answered]
I've hit this occasionally where an event will not complete but stays tagged as "in progress".

Latest example was with an attic "rodent cam". The video clip starts out with a moth whizzing past the camera. That seems to trigger the start of the event, but it never closes. So my question is what "times out" an event, i.e. "nothing else has happened so we're closing this one out"... ?? Is there a config option?

Running -  Frigate (0.12.0-da3e197), two Coral pcie boards configured as detectors.

    Attic: # camera name
        motion:
            mask:
            - 1280,0,1280,720,687,720,477,0
        ffmpeg:
            inputs:
                - path: rtsp://<my info>
                  roles:
                    - record
                    - detect
            input_args: preset-rtsp-restream
        rtmp:
            enabled: False
        detect:
            width: 1280 #
            height: 720
        record:
            enabled: True
            retain:
                days: 3
                mode: all
            events:
                retain:
                    mode: active_objects
                    default: 20
        snapshots:
            enabled: True
            retain:
                default: 20
        live:
            stream_name: AtticLive

**Comment by NickM-27** [comment]
This usually happens for to oversized motion masks causing objects to get stuck

**Reply by FrigateEnthusiast** [reply]
Any guidelines as to what would/could make a mask "oversized"?

**Reply by NickM-27** [reply, marked as answer]
Masks are only supposed to be used to block out things like Timestamps, trees, the sky, etc. 

Usually they're too big simply because they're blocking more than that, but not always. https://docs.frigate.video/configuration/masks

**Reply by FrigateEnthusiast** [reply]
I would consider this a bug. This condition should be detected or there should be some type of timeout or watchdog after an object tracking starts.

**Reply by NickM-27** [reply]
The behavior with motion masks could be improved but frigate is designed to have an event last as long as the object is visible in the frame. Given that a large motion masks forces frigate to ignore the motion of the object leaving, it sees the object as still there.

**Reply by FrigateEnthusiast** [reply]
How about a right-click where the user could manually end/reset any in progress tracking sequences?

**Reply by NickM-27** [reply]
Like I said we are looking to improve the behavior of objects getting caught in motion masks. Other than that there are no valid reasons to end in progress events since they are true in progress events.
